.github/workflows/codegen-check.yaml

```
@@ -0,0 +1,32 @@
+# Copyright 2023 SECO Mind Srl
+# SPDX-License-Identifier: Apache-2.0
+
+name: codegen-check
+
+permissions:
+  contents: read
+
+on:
+  workflow_call:
+
+env:
+  CARGO_TERM_COLOR: always
+  PB_REL: https://github.com/protocolbuffers/protobuf/releases
+  PB_VERSION: 24.3
```

Bump to the latest version
```suggestion
  PB_VERSION: 24.4
```

---

Done